iospi.yml

```diff
@@ -0,0 +1,19 @@
+name: iospi
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.9
```

Comment:
NIT: I think it is good practice to put the requirements in alphabetic order: so that we do not import the same ones twice, and so that it is easy to find the one we need!

Reply:
Ah yes, thanks for pointing this out! Now corrected :)